src/Core/src/Handlers/SwipeItemMenuItem/ISwipeItemMenuItemHandler.cs

```diff
@@ -16,5 +16,6 @@ public partial interface ISwipeItemMenuItemHandler : IElementHandler
 	{
 		new ISwipeItemMenuItem VirtualView { get; }
 		new PlatformView PlatformView { get; }
+		ImageSourcePartLoader SourceLoader { get; }
```

Comment:
This is a new API to both be consistent and to avoid concrete type casts in the mappers which break all users of that mapper.

Reply:
do we want this to be nullable and use a default interface method?

Reply:
good catch.